zmsdldb/src/Zmsdldb/TwigExtension.php

```
@@ -104,15 +104,15 @@ public function getD115Enabeld()
     public function getD115OpeningTimes()
     {
         $settingsRepository = \App::$repository->fromSetting();
-        $openinTimes = $settingsRepository->fetchName('d115.openingTime') ?? \APP::D115_DEFAULT_OPENINGTIME;
+        $openinTimes = $settingsRepository->fetchName('d115.openingTime') ?? \App::D115_DEFAULT_OPENINGTIME;
```

PHPStan (2.2.7)

[error] Access to undefined constant App::D115_DEFAULT_OPENINGTIME.


(classConstant.notFound)